_data/simple-icons.json

```
@@ -5534,6 +5534,11 @@
             "hex": "005A2B",
             "source": "https://media.landrover.com/en/press-kit"
         },
+        {
+            "title": "Lapce",
+            "hex": "3B82F6",
+            "source": "https://github.com/lapce/lapce"
```

Could you please change the source URL to the SVG file in that repo and as a permalink? We've recently changed the contribution guidelines regarding the source URL when the SVG icon in a GitHub repository does not contain the brand color (https://github.com/simple-icons/simple-icons/pull/7357).

```suggestion
            "source": "https://github.com/lapce/lapce/blob/95c4cf2d87083e348c0b621d0be0ea17f79ed703/extra/images/logo.svg"
```